### PR TITLE
Multi cluster with federation

### PR DESCRIPTION
**Pull request by cloudbehl**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -22,6 +22,7 @@ PROMETHEUS_ALERTS_DIR=monitoring/ceph-mixin
 #PROMETHEUS_ALERTS_DIR=monitoring/prometheus/alerts
 
 PROMETHEUS_HOST_PORT=9090
+PROMETHEUS_HOST_PORT1=9098
```
Comment (nizamial09) [inline .env.example]:
```suggestion
PROMETHEUS2_HOST_PORT=9098
```

Comment (nizamial09) [reply .env.example]: and does this needs to be different since prometheus2 running on a separate container?

**Review by nizamial09 — COMMENTED** [review]
Locally works! thank you

File: .env.example (hunk 1 above)
Comment (cloudbehl) [reply .env.example]: Yes it needs to be different as this is mapping to local nodes ip. If we don’t change this will conflict

**Review by nizamial09 — APPROVED** [review]

**Pull request merged**